zebra/rt_netlink.c

```diff
@@ -683,7 +683,7 @@ static int netlink_route_change_read_multicast(struct nlmsghdr *h,
 	if (tb[RTA_DST])
 		m->sg.grp = *(struct in_addr *)RTA_DATA(tb[RTA_DST]);
 
-	if ((RTA_EXPIRES <= RTA_MAX) && tb[RTA_EXPIRES])
+	if (tb[RTA_EXPIRES])
```

Comment:
This must not be removed!  If we are compiling on older versions of linux this will cause a crash and burn.